census/views.py

```diff
@@ -53,8 +54,46 @@ def events_of(size=1000):
 
 
 def index(request):
+    """
+    Homepage
+    """
+    # Uncomment this if we need to call the homepage in the future using AJAX
+    # if request.is_ajax():
+    #     return get_events()
+    events = models.Event.objects.all()
+    request.events = events
+    # current_date = datetime.now()
+    # request.events_title = f'{current_date.strftime("%B")} {current_date.strftime("%Y")} Events'
     return render(request, 'index.html')
 
+
+def get_events(data):
+    """
+    This endpoint /events is called by the datepicker (census/static/datepicker/js/datepicker.js)
+    to retrieve a list of filtered events by selected date/month.
+
+    :param data: Request object
+    :return: HttpResponse object containing event data. Populate
+    """
+    query_params = data.GET.dict()
+    if 'isMonthly' in query_params and query_params['isMonthly'] == 'true':
+        # get monthly events
+        # TODO: Fetch Event objects based on start and end dates
+        month = query_params['month']
+        return HttpResponse(json.dumps(dict(request={'events': [
+            dict(id=4, title='Title4', description='SSS', start_time='123', end_time='433'),
```

Comment:
This is dummy data for now. I will be making changes here to actually filter events based on date when the PR to add start and end dates back to the model is merged.